app/screens/channel/channel_post_list/channel_post_list.tsx

```diff
@@ -44,28 +45,47 @@ const ChannelPostList = ({
     const isTablet = useIsTablet();
     const serverUrl = useServerUrl();
     const canLoadPosts = useRef(true);
-    const fetchingPosts = useRef(false);
-
+    const [fetchingPosts, setFetchingPosts] = useState(EphemeralStore.isLoadingMessagesForChannel(serverUrl, channelId));
```

Comment:
This is a state now controlled by the EphemeralStore through the `LOADING_CHANNEL_POSTS` event. That way we can better control when we fetch more messages.